content/3_Silver/Flood_Fill.mdx

```diff
@@ -235,23 +241,42 @@ int main(){
 <JavaSection>
 
 ```java
-static int[][] grid;
-static int n, m;
-
-public static void main(String[] args){
-	int n = r.nextInt();
-	int m = r.nextInt();
-	grid = new int[n][m];
-	for(int i = 0; i < n; i++){
-		for(int j = 0; j < m; j++){
-			grid[i][j] = r.nextInt();
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Floodfill {
+	private static int[][] grid;
+	private static int rowNum;
+	private static int colNum;
+	public static void main(String[] args) throws IOException {
+		BufferedReader read = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer dims = new StringTokenizer(read.readLine());
+		rowNum = Integer.parseInt(dims.nextToken());
+		colNum = Integer.parseInt(dims.nextToken());
+		grid = new int[rowNum][colNum];
+		for (int r = 0; r < rowNum; r++) {
+			grid[r] = Arrays.stream(read.readLine().split(" "))
```

Comment:
I'd prefer not to use stream esp for silver :/.

Reply:
Would adding a short explanation comment work?

Reply:
Just don't use stream